### ISSUE TITLE
Switch winapi dependency to windows/windows-sys

### ISSUE THREAD

**Issue by atouchet** [mozilla/cargo-vet#643]
The winapi crate is no longer maintained and there are now official windows/windows-sys crates from Microsoft. cargo-vet should transition from using winapi to windows/windows-sys.

**Comment by berlin2424** [comment]
Estimados, esta persona me tiene hackeado mis dispositivos, favor necesito ayuda ya que vio que lo descubrí y empezó a modificar lo que tenia programado su nombre anterior era Rminnich y antes de ese se tenía andreswsun2898 y ahora acaba de cambiarse el nombre a toque

**Comment by mrghosti3** [comment]
Just checked [mozilla registry](https://raw.githubusercontent.com/mozilla/supply-chain/main/audits.toml) in `registry.toml` and found that it contains entry for `windows-sys`:

```
  8919	[[trusted.windows-sys]]
  8920	criteria = "safe-to-deploy"
  8921	user-id = 64539
  8922	start = "2021-11-15"
  8923	end = "2024-09-12"
  8924	aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
  8925	
```

Does this solve your issue?

**Comment by atouchet** [comment]
No, this is about cargo-vet still using winapi as a dependency:

https://github.com/mozilla/cargo-vet/blob/892f3bc8b1d8173f1768868599b6dfcd2c772bf5/Cargo.toml#L56-L63

**Comment by mrghosti3** [comment]
Ah, fair, misread the issue. Sorry.

**Comment by mystor** [comment]
In #658, the direct dependency from `cargo-vet` on `winapi` has been changed to use `windows-sys`. `cargo vet` still depends on `winapi` due to some dependencies which still depend on that crate (see the `Cargo.lock` if you're curious about which dependencies).
